src/v/redpanda/application.cc

```diff
@@ -1576,6 +1576,20 @@ void application::wire_up_redpanda_services(model::node_id node_id) {
               };
 
               c.connection_rate_bindings.emplace(std::move(bindings));
+
+              net::tcp_keepalive_bindings keepalive_bindings{
+                .keepalive_idle_time
+                = config::shard_local_cfg()
+                    .kafka_tcp_keepalive_idle_timeout_seconds.bind(),
+                .keepalive_interval
+                = config::shard_local_cfg()
+                    .kafka_tcp_keepalive_probe_interval_seconds.bind(),
+                .keepalive_probes
+                = config::shard_local_cfg().kafka_tcp_keepalive_probes.bind(),
+
+              };
+
+              c.tcp_keepalive_bindings.emplace(std::move(keepalive_bindings));
```

Comment:
I guess it's a matter of style but couldn't you just do

```
net::tcp_keepalive_bindings  = keepalive_bindings{ ... // same as above
```

? 

Avoids the need to create the named object, then immediately move it out into the object.

Reply:
No other reason than copying existing code but that's usually not a good reason. Fixed.